enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -2222,6 +2222,20 @@ func (c *FirecrackerContainer) Exec(ctx context.Context, cmd *repb.Command, stdi
 		}
 	}()
 
+	// Emit metrics to track time spent preparing VM to execute a command
+	if c.memoryStore != nil {
+		c.memoryStore.EmitUsageMetrics("init")
+	}
+	if c.uffdHandler != nil {
```

Comment:
There is some redundancy here between memoryStore and uffdHandler metrics, but I was curious how much total time was spent handling page faults including the UFFDIO_COPY etc.